src/interfaces.ts

```diff
@@ -122,41 +122,22 @@ export enum DefaultEditorId {
   'css' = 'styles.css',
 }
 
-export type CustomEditorId = Exclude<
-  `${string}.${'js' | 'html' | 'css'}`,
-  DefaultEditorId
->;
+export type CustomEditorId = `${string}.${'js' | 'html' | 'css'}`;
 
-export type EditorId = DefaultEditorId | CustomEditorId;
-
-// Panels that can show up as a mosaic
-export const enum PanelId {
-  'docsDemo' = 'docsDemo',
-}
-
-export type MosaicId = DefaultEditorId | CustomEditorId | PanelId;
+export type EditorId = CustomEditorId | DefaultEditorId;
 
 export const DEFAULT_EDITORS = [
   DefaultEditorId.main,
   DefaultEditorId.renderer,
   DefaultEditorId.preload,
   DefaultEditorId.html,
   DefaultEditorId.css,
-];
+] as const;
```

Comment:
can we consolidate this with `SORTED_EDITORS`?

Reply:
Oh I like that idea! Lemme try it out...

Reply:
added in 33586bc9ca6e23dc73bcfb38c9b46c6a37f36688